include/swift/Subsystems.h

```diff
@@ -264,6 +264,17 @@ namespace swift {
   void serialize(ModuleOrSourceFile DC, const SerializationOptions &options,
                  const SILModule *M = nullptr);
 
+  /// Serializes a module or single source file to the given output file and
+  /// returns back the file's contents as a memory buffer.
+  /// Use this if you intend to immediately load the serialized module, as that
```

Comment:
Docs nitpick: this would be a good place for a newline to separate the "short" and "full" docs.